docs/user_guide/HOWTO/NFS.rst

```diff
@@ -124,49 +137,48 @@ default.yaml
 
 .. code-block:: yaml
 
-  nfs::server:             "your.server.fqdn"
-  nfs::server::client_ips: "%{alias('client_nets')}"
-  nfs::simp_iptables:      true
-  nfs::simp_krb5:          false
+  nfs::is_server: false
+  nfs::client::stunnel::nfs_server: <your nfs server>
+  simp_nfs::home_dir_server: <your nfs server>
```

Comment:
Inconsistent references to fqdn's.  Perhaps would be easiest if in the beginning, of the readme, we say that the code below assumes the NFS server is nfs.server.your.domain and NFS client is nfs.client.your domain.  Inconsistent naming is confusing to newbies like me.